### ISSUE TITLE
Add instructions for how to install Ubuntu & Debian / Red Hat & CentOS versions of OSS 2.x

### ISSUE THREAD

**Issue by danxmoran** [influxdata/docs-v2#1991]
The "Installing InfluxDB OSS" page for OSS 1.x includes the commands needed to install InfluxDB from repos.influxdata.com via `apt` / `yum`. v2.0.3 revamped our packaging strategy for OSS 2.x and published new archives to the repo, so we should update the 2.x docs to match.

The 2.x installation instructions for deb & rpm should be identical to the 1.x version, _except_ that we've renamed the package from "influxdb" to "influxdb2". This change should only affect the final `apt-get install` / `yum install` commands.

##### Relevant URLs
- V1 instructions: https://docs.influxdata.com/influxdb/v1.8/introduction/install/#installing-influxdb-oss

**Comment by alsargent** [comment]
Agreed. The [InfluxDB OSS 2.0 page](https://docs.influxdata.com/influxdb/v2.0/get-started/?t=Linux) should list how to install AND upgrade using `apt` (for Debian/Ubuntu) `yum` (for RHEL/Centos), and `brew` (for macOS). Assuming, of course, that all those package managers will install InfluxDB OSS 2.0. (If they don't, those are separate issues.)

[This tweet](https://twitter.com/ascherbaum/status/1374370225135620097) is from a user that has run into this issue: 

![image](https://user-images.githubusercontent.com/20701/112247312-539c0a80-8c11-11eb-9bf5-95c9535082cd.png)

**Comment by alsargent** [comment]
@sanderson Not sure if you're the right person to assign this to, but thought I'd start with you. Feel free to unassign yourself and/or reassign to someone else. Thanks.

**Comment by andreasscherbaum** [comment]
@alsargent Thanks for bringing this up.

With the current set of install instructions on the website it is not possible to automatically install v2.
One has to go to the website, find the specific link, then download a single package.
And this does not include automatic updates (like bugfixes and such).
